### PR TITLE
Update summer19

### PR DESCRIPTION
**Pull request by nileshpatra**
updated summer'19 index.html, added project link and description.

### CODE REVIEW

File: summer19/index.html [hunk 1]
```diff
@@ -14,19 +14,31 @@
...
+We bet, this will be an amazing learning experience which will also leave its mark on your open source profile ;) We have planned to make a meetup automation api and make webapp and android apps for the same. 
+These are the modules that the project has been broken down into:
 
-Do fill the form below, suggest some awesome project ideas and get a chance to implement them ;). We will finalize it soon, and for people starting out in open source, this is going to be the best starting point. These are the roles which you can take up :
+<p>
+<ul>
+	<li><p><a href="https://github.com/ILUGD/Birdman" style="color:red" target="blank">Birdman NotASPAMbox</a> - Automated mail and social media publishing command line interface</p></li>
+
+	<li><p><a href="https://github.com/ILUGD/Splatter" style="color:red" target="blank">Splatter</a> - Poster generator in Golang</p></li>
+	<li><p><a href="https://github.com/ILUGD/valkommen" style="color:red" target="blank">Välkommen</a> - automated attendance and feedback retrieval system</p></li>
```
Comment (hellozee) [inline summer19/index.html]:
```suggestion
	<li><p><a href="https://github.com/ILUGD/valkommen" style="color:red" target="blank">Välkommen</a> - Automated attendance and feedback retrieval system</p></li>
```

File: summer19/index.html [hunk 2]
```diff
@@ -14,19 +14,31 @@
...
+We are hosting a bunch of community projects for the summer. If you’re in for some real challenge this holiday season and commit some time to solve a problem with open source solution, the time has come !! 
+We bet, this will be an amazing learning experience which will also leave its mark on your open source profile ;) We have planned to make a meetup automation api and make webapp and android apps for the same. 
+These are the modules that the project has been broken down into:
 
-Do fill the form below, suggest some awesome project ideas and get a chance to implement them ;). We will finalize it soon, and for people starting out in open source, this is going to be the best starting point. These are the roles which you can take up :
+<p>
+<ul>
+	<li><p><a href="https://github.com/ILUGD/Birdman" style="color:red" target="blank">Birdman NotASPAMbox</a> - Automated mail and social media publishing command line interface</p></li>
+
+	<li><p><a href="https://github.com/ILUGD/Splatter" style="color:red" target="blank">Splatter</a> - Poster generator in Golang</p></li>
```
Comment (hellozee) [inline summer19/index.html]:
```suggestion
	<li><p><a href="https://github.com/ILUGD/Splatter" style="color:red" target="blank">Splatter</a> - Automated poster generator</p></li>
```

Comment (nileshpatra) [reply summer19/index.html]: did both the changes, please check, :)

**Comment by hellozee** [comment]
LGTM

**Pull request merged**